docs/formats.md

```diff
@@ -33,8 +33,7 @@ There is an extensive (but not exhaustive) list of [included formats](#pre-defin
 
 ### Format configuration
 
-Formats can take configuration to make them more flexible. This allows you to re-use the same format multiple times with different configuration or to allow the format to use data not defined in the tokens themselves. To configure a format add extra attributes on the file object in your configuration like so:
-
+Formats can take configuration to make them more flexible. This allows you to re-use the same format multiple times with different configuration or to allow the format to use data not defined in the tokens themselves. To configure a format add extra attributes on the file object in your configuration like following:
```

Comment:
I believe this needs a "the" before "following"

Reply:
OK, @dbanksdesign . Added some changes too.